packages/core-mobile/app/hooks/earn/useImportAnyStuckFunds.ts

```diff
@@ -20,21 +23,32 @@ export const useImportAnyStuckFunds = (
   const activeAccount = useSelector(selectActiveAccount)
   const isDevMode = useSelector(selectIsDeveloperMode)
   const selectedCurrency = useSelector(selectSelectedCurrency)
+  const activeNetwork = useSelector(selectActiveNetwork)
+  const { getFeeState } = useGetFeeState()
+  const feeState = getFeeState()
 
```

Comment:
I think this one needs useMemo

Reply:
actually, I don't think it needs to be out here and you can probably move it inside queryFn. you should also move isDevnet(activeNetwork) outside so you only need to do `['ImportAnyStuckFunds', activeAccount, isDevMode, isDevnet]` as the queryKey